core/src/main/java/org/infinispan/persistence/manager/PersistenceManagerImpl.java

```diff
@@ -585,7 +621,12 @@ private void initializeWriter(StoreConfiguration cfg, CacheWriter writer, Initia
       if (writer != null) {
          if (writer instanceof DelegatingCacheWriter)
```

Comment:
So does this mean that for example a singleton writer won't participate in the transaction?